docs/source/network/network.md

```diff
@@ -1,1032 +1,165 @@
-# Blockchain network
+# How blockchain networks are structured
```

Comment:
I am a sure that there has been discussion about this, but should this article be about how Fabric networks are structured. Even if blockchain = Fabric in the Fabric docs, I think it would help to have the title be more precise.

Reply:
Nice idea. I like it.